avm/res/batch/batch-account/main.json

```diff
@@ -1486,4 +1486,4 @@
       "value": "[if(and(coalesce(tryGet(parameters('managedIdentities'), 'systemAssigned'), false()), contains(reference('batchAccount', '2022-06-01', 'full').identity, 'principalId')), reference('batchAccount', '2022-06-01', 'full').identity.principalId, '')]"
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
Appears unintended. Please undo.

Reply:
Thanks, have reverted this file back to upstream/main